tests/unit/-private/properties/collection-test.js

```diff
@@ -214,7 +214,7 @@ moduleForProperty('collection', function(test) {
 
     this.adapter.createTemplate(this, page);
 
-    assert.throws(() => page.foo.bar.objectAt(1).baz.qux, /page\.foo\.bar\[1\]\.baz\.qux/);
+    assert.throws(() => page.foo.bar.objectAt(1).baz.qux, /page\.foo\.bar\[1\]/);
```

Comment:
We just need to update the `test` text to say something like `'throws an error when the item's element doesn't exist'`.